pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/executor/OpChainSchedulerService.java

```diff
@@ -98,9 +108,12 @@ public void runJob() {
                 // not complete, needs to re-register for scheduling
                 register(operatorChain, false);
               } else {
-                LOGGER.debug("({}): Completed {}",
-                    operatorChain,
-                    operatorChain.getStats());
+                if (result.isErrorBlock()) {
+                  LOGGER.error("({}): Completed erroneously {} {}", operatorChain, operatorChain.getStats(),
+                      result.getDataBlock().getExceptions());
+                } else {
+                  LOGGER.debug("({}): Completed {}", operatorChain, operatorChain.getStats());
```

Comment:
irrelevant to this PR. but complete ing of an entire opchain. can we do a INFO log? (this is once per stage yes?)

Reply:
I feel like that's a lot of logging, are we sure we're OK with that? I guess we can always disable it if we want

Reply:
it's the only debug log in the class, so we can always turn on debug just for this class if we want to in prod. WDYT?